alt_linkedlist_palindrome.js

```diff
@@ -0,0 +1,64 @@
+//check if linked list is palindrome
+//linked list
+// - chain of nodes
+
+class LinkedList {
+    //create new node
+    constructor(next,data) {
```

Comment:
To avoid having to pass `null` explicitly to your constructor (which is a very Java thing), you can switch the order of your parameters. JS will assign `undefined` to parameters by default if an argument doesn't get passed to it.
```suggestion
    constructor(data, next) {
```